gen3cirrus/aws/manager.py

```diff
@@ -0,0 +1,7 @@
+class AwsCloudManager:
```

Comment:
Let's just remove this. I don't see a clear need going forward and we can always add it later